src/handlers/feature_handlers.cpp

```diff
@@ -6645,15 +6645,30 @@ void vehSirenHandler::Install() {
         }
     );
 
-    InstallCallback("vehSiren::SizeOf", "Change size of vehSiren on vehicle initialization.",
-        &SizeOf, {
+    InstallCallback("vehCar::Init", "Use our new size of vehSiren on vehicle initialization.",
+        &OperatorNew, {
             cb::call(0x42BE30),
         }
     );
 
-    //jmp out ltLightPool destructor
-    InstallPatch({ 0xEB }, {
-        0x4D6638,
+    //set old size of vehSiren to 0
```

Comment:
This patch doesn't really do anything, since you are hooking the subsequent `operator new` call, and ignoring the parameter it corresponds to.